README.md

```diff
@@ -21,6 +21,19 @@ Calico features:
 
 Check out our [wiki pages](https://github.com/yangjames/Calico/wiki) for more info.
 
+# Dependencies
+
+* Calico is not compatible with Ubuntu 20.04 the minimum supported version is 22.04.
+* Install the latest version of the CUDA sdk
```

Comment:
What results in the CUDA dependency? Calico doesn't use anything particular within OpenCV to require GPU.

Reply:
seems to be needed by ceres on my machine, I tried setting `USE_CUDA=false` but it insisted on cuda anyway. IDK why.